internal/payloads/reservation_payload.go

```diff
@@ -68,6 +68,9 @@ type AWSReservationResponsePayload struct {
 
 	// Immediately power off the system after initialization
 	PowerOff bool `json:"poweroff"`
+
+	// Instance IDs, only present for finished reservations
+	Instances []string `json:"instances,omitempty"`
```

Comment:
Why do we want/need to add this in the payload? When I want to list reservations do we want to see all the instances IDs that were created?

Reply:
Fixes: https://issues.redhat.com/browse/HMSPROV-254